build/AzurePipelineTemplates/CsWinRT-BuildAndTest-Stage-GitHub.yml

```diff
@@ -0,0 +1,106 @@
+stages:
+- stage: BuildAndTest
+  displayName: Build/Test Stage
+  jobs: 
+  - job: BuildAndTest
+    timeoutInMinutes: 270

```

Comment:
3 hours?😮

Reply:
At some point, we used to hit that when we built from TFMs at the same time.  Lately it has been faster.  At some point, might make sense to reduce.